File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int w = 0;
        int h = 0;
        
        for(int i = 0; i < sizes.length; i++){
            int a = sizes[i][0] > sizes[i][1] ? sizes[i][0] : sizes[i][1];
            int b = sizes[i][0] > sizes[i][1] ? sizes[i][1] : sizes[i][0];

            if(a > w){
                w = a;
            }
            if(b > h){
                h = b;
            }
        }
        
        answer = w * h;
        
        return answer;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    w, h = 0, 0

    for a, b in sizes:
        if a < b:       #b가 a보다 클때, 가장 큰 값을 넣는 a로 변경
            a, b = b, a
        w = max(w, a)
        h = max(h, b)
            
    answer = w * h
    
    return answer